docs/requestCourse.md

```diff
@@ -2,9 +2,9 @@
 
 To request your PrairieLearn course space, you can use the "Request course" links on the PrairieLearn home page or in the main dropdown menu.
 
-![](requestCourseButton.png)
-![](requestCourseDropdown.png)
+![](img/requestCourseButton.png)
+![](img/requestCourseDropdown.png)
 
 This will bring you to a web form where you can request your course.
 
-This would also be a good opportunity to join the PrairieLearn slack, if you haven't already: [https://prairielearn.com/slack](https://prairielearn.com/slack)
+This would also be a good opportunity to join the PrairieLearn slack, if you haven't already: [:fontawesome-brands-slack: https://prairielearn.com/slack](https://prairielearn.com/slack)
```

Comment:
Maybe tell people why they might want to join Slack?

```suggestion
This would also be a good opportunity to join the PrairieLearn Slack, if you haven't already: [:fontawesome-brands-slack: https://prairielearn.com/slack](https://prairielearn.com/slack)
```